.gitlab-ci.yml

```diff
@@ -155,6 +155,15 @@ yarn lint:
     - apk add --no-cache git
     - yarn lint
 
+translations lint:
+  image: alpine
```

Comment:
Using an explicit version would be advised. But considering none of the images are pinned to a version *shrug*